crates/samples/consent/src/main.rs

```diff
@@ -2,7 +2,7 @@ use windows::{core::*, Foundation::*, Security::Credentials::UI::*, Win32::Found
 
 fn main() -> Result<()> {
     unsafe {
-        let interop = factory::<UserConsentVerifier, IUserConsentVerifierInterop>()?;
+        let interop = factory::<UserConsentVerifier, IUserConsentVerifierInterop>(None)?;
```

Comment:
Remove the `None` as that's not going to be norm so we don't want folks to have to `None` all the things. But would be good to add a test for `from_library` if there's a system DLL that would work that way...